pegasus/sites.v3/code.org/public/educate/applab2.haml

```diff
@@ -0,0 +1,61 @@
+---
+title: App Lab
+theme: responsive
+video_player: true
+---
+
+%link{href: "/css/jumbotron-banner.css", rel: "stylesheet"}
+%link{href: "/shared/css/course-blocks.css", rel: "stylesheet"}
+%link{href: "/css/highlights.css", rel: "stylesheet"}
+
+= view :jumbotron_banner, img: CDO.code_org_url("/images/app-lab/girl-cropped.jpg"), title: "App Lab", desc: "App Lab is a web-based environment for students to make simple apps. Simply use \"design mode\" to design your app, write code for you app with blocks or in JavaScript, then send the app to your phone in seconds.", specs: "Grades 9-12, All modern browsers, English only", url: CDO.studio_url('/p/applab'), cta: "Make an app", nav_bar: [{text: 'Resources', url: '#resources'}, {text: 'Challenges', url: '#challenges'}, {text: 'Video Library', url: '#videos'}]
+
+#resources.responsive-body
+  %h1 Resources to get you started
+  Code.org has created a new programming tool, called <a href="#{CDO.studio_url('/p/applab')}">App Lab</a> for the new <a href="#{CDO.code_org_url('/educate/csp')}">Computer Science Principles</a> high school course. You do not need to complete the Computer Science Principles course to use App Lab, but if you'd like an introduction to the tool here are some resources to help.
```

Comment:
oh, i see that for i18n you probably want to do it this way.  alternative would be special tag substitutions.